R/experiment.R

```diff
@@ -124,23 +150,35 @@ validate_experiment <- function(x) {
   colnames <- lapply(c(parameters, obs_rates), function(f) names(f(x)))
 
   check_experiment(name(x), model)
-  test_schar(names(dic_g2r))
+  # test_schar(names(dic_g2r)) check if it works in different gama versions
```

Comment:
We can remove it as for the display we need the last version of GAMA and in the last version, we don't have this problem of special character in the name of parameters.

Reply:
Done in commit `2f48cdf`